doc/design/architecture.md

```diff
@@ -11,17 +11,19 @@ Each of these Operators is responsible for managing the CRDs that are the basis
 | CatalogSource         | catsrc         | Catalog | a repository of CSVs, CRDs, and packages that define an application                        |
 | Subscription          | sub        | Catalog | used to keep CSVs up to date by tracking a channel in a package                            |
 | OperatorGroup         | og     | OLM     | used to group multiple namespaces and prepare for use by an operator                     |
+| PackageManifest       |        | PackageServer | provides the user with an api presentation of the data a CatalogSource provides |
 
 Each of these Operators are also responsible for creating resources:
 
-| Operator | Creatable Resources        |
-|----------|----------------------------|
-| OLM      | Deployment                 |
-| OLM      | Service Account            |
-| OLM      | (Cluster)Roles             |
-| OLM      | (Cluster)RoleBindings      |
-| Catalog  | Custom Resource Definition |
-| Catalog  | ClusterServiceVersion      |
+| Component | Creatable Resources        |
+|-----------|----------------------------|
+| OLM       | Deployment                 |
+| Catalog   | Service Account            |
+| Catalog   | (Cluster)Roles             |
+| Catalog   | (Cluster)RoleBindings      |
+| Catalog   | Custom Resource Definition |
+| Catalog   | ClusterServiceVersion      |
+| Package Server | PackageManifest       |
```

Comment:
Same comment here, you can't actually create a packagemanifest resource. It needs to be derived from a catalog.

Reply:
I would recommend to change the word "creatable" to "Managed"

Reply:
I dealt with the first table by removing it and adding a paragraph below on the PackageManifest so it is close, but clear that it isn't the same. I also added @dinhxuanvu's word change idea.